src/H5Fpublic.h

```diff
@@ -189,10 +189,11 @@ typedef enum H5F_libver_t {
     H5F_LIBVER_V110     = 2, /**< Use the latest v110 format for storing objects */
     H5F_LIBVER_V112     = 3, /**< Use the latest v112 format for storing objects */
     H5F_LIBVER_V114     = 4, /**< Use the latest v114 format for storing objects */
+    H5F_LIBVER_V116     = 5, /**< Use the latest v116 format for storing objects */
     H5F_LIBVER_NBOUNDS       /**< Sentinel */
 } H5F_libver_t;
 
-#define H5F_LIBVER_LATEST H5F_LIBVER_V114
+#define H5F_LIBVER_LATEST H5F_LIBVER_V116
```

Comment:
Caught another 1.16 issue, there are more that need to be found.

Reply:
Fixed in next PR